partcad/src/partcad/wrappers/custom_cqgi.py

```
@@ -0,0 +1,619 @@
+                .add_entry("debug", collector.debug)
+                .add_entry("describe_parameter", collector.describe_parameter)
+                .build()
+            )
+
+            c = compile(self.ast_tree, CQSCRIPT, "exec")
+            exec(c, env)
+            result.set_debug(collector.debugObjects)
+            result.set_success_result(collector.outputObjects)
+            result.env = env
+
+        except Exception as ex:
```

Ruff (0.8.2)

Do not catch blind exception: `Exception`

(BLE001)